cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -80,4 +80,14 @@ function up() {
             exit 1
         fi
     fi
+
+    if [ "$KUBEVIRT_DEPLOY_CDI" == "true" ]; then
+        $kubectl create -f /opt/cdi-*-operator.yaml
+        $kubectl create -f /opt/cdi-*-cr.yaml
+        while [ $($kubectl get pods --namespace cdi | grep 'cdi-' | wc -l) -lt 4 ]; do
+            $kubectl get pods --namespace cdi
+            sleep 10
+        done
+        until $kubectl wait --for=condition=Ready pod --timeout=60s --all --namespace cdi; do sleep 10; done
```

Comment:
maybe we want to make sure this doesnt drift forever in case of an error ? (according lane max time)
not a must

Reply:
@oshoval I think the timeout is already defined in the command in the flag `--timeout=60s`

Reply:
it is until forever loop in case the command fails, because the do part is conditionless
it will just sleep and retry
the timeout is per try, which act as the condition